pages/_app.js

```diff
@@ -1,5 +1,6 @@
 import GlobalStyle from "../styles";
 import { SWRConfig } from "swr";
+import Header from "@/components/header";
```

Comment:
By convention, React component names should be written in PascalCase (e.g., Header.js, Card.js, CardList.js). This ensures consistency with the component function names.

Reply:
Thanks, done.